src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let result = '';
+  const dateParts = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
Consider using destructuring to assign date parts to the newDate object directly.